tailwind.config.js

```diff
@@ -0,0 +1,75 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    "./pages/**/*.{js,ts,jsx,tsx}",
+    "./components/**/*.{js,ts,jsx,tsx}",
+  ],
+  theme: {
+    fontSize: {
+      h1: rem(68),
+      h2: rem(50),
+      h3: rem(42),
+      h4: rem(38),
+      h5: rem(28),
+      h6: rem(22),
+      sh1: rem(22),
+      b1: rem(20),
+      b2: rem(16),
+      b3: rem(14),
+      b4: rem(12),
+      c2: rem(9),
+      c3: rem(6),
+    },
+    lineHeight: {
+      h1: "calc(90 / 68)",
+      h2: "calc(60 / 50)",
+      h3: "calc(55 / 42)",
+      h4: "calc(50 / 38)",
+      h5: "calc(38 / 28)",
+      h6: "calc(30 / 22)",
+      sh1: "calc(30 / 22)",
+      b1: "calc(30 / 20)",
+      b2: "calc(24 / 16)",
+      b3: "calc(17 / 14)",
+      b4: "calc(17 / 12)",
+      c2: "calc(19 / 9)",
+      c3: "calc(17 / 6)"
+    },
+    letterSpacing: {
+      heading: "0em",
+      copy: "0.01em",
+    },
+    fontWeight: {
+      400: 400,
+      450: 450,
+      600: 600,
+      800: 800
```

Comment:
we have one outlier weight (450), so i overrode the defaults (medium, semibold, etc) with these numbered weights for clarity, and only these should show up in autocomplete